schema.sql

```diff
@@ -0,0 +1,52 @@
+create schema if not exists appdata;
+
+CREATE TABLE IF NOT EXISTS appdata.user
+(
+  user_id bigserial NOT NULL 
+  , email varchar(50) UNIQUE NOT NULL
+  , password varchar(10) NOT NULL
+  , name varchar(30) NOT NULL
+  , age smallint 
+  , gender varchar(3)
+  , city varchar(20)
+  , country varchar(20)
+  , browser varchar(10)
+  , is_developer boolean
+  , created_on timestamp without time zone NOT NULL
+  , last_login timestamp without time zone NOT NULL
+  , PRIMARY KEY (user_id)
+
+ );
+
+
+CREATE SEQUENCE if not exists poll_image_id_seq;
```

Comment:
Like I mentioned above, I still feel it is easier to control id sequence in python :). Nonetheless, great thinking and awesome work!

Reply:
lgtm